Problem_3.cpp

```diff
@@ -0,0 +1,18 @@
+class Solution {
+public:
+    int findPeakElement(vector<int>& nums) {
+        if (nums.size() == 1) return 0;
+        int left = 0;
+        int right = size(nums) - 1;
+
+        while(left <= right){
```

Comment:
because of = sign mid is assigned infinite times to right  and time limit is exceeding